docs/configuration/traces-config.md

```diff
@@ -1,11 +1,11 @@
 +++
-title = "tempo_config"
+title = "traces_config"
```

Comment:
I haven't done this for logs/metrics yet because I'm not sure how to have this alias the old URL so old links continue to work. Can you look into how to do that? @robbymilo may be able to help here.

Reply:
Looks like robby is on vacation, added hugo style but can roll them back and add that alias in a later commit once we verify it works